### ISSUE TITLE
Add an ML Algorithm for classification of the admission data

### ISSUE THREAD

**Issue by Sohammhatre10** [Sohammhatre10/Cadmi_AI#4]
Requirements are -

1. Create an ML Algorithm using `Scikit-learn` , `Tensorflow` or `PyTorch` for the classification of the scrapped admission data.
2. Provide users with predictive analysis of admissions in his/her desired field.
3. You may use Regression algorithms on Kaggle or Huggingface for the same if necessary.

**Comment by gaurav-rm11** [comment]
assign me

**Comment by Sohammhatre10** [comment]
Will assign this after the scrappers are successful. Till then this will be open to all.

**Comment by abhirajadhikary06** [comment]
If we could take the instantly generated csv files and create a prediction and an analysis chat out of those and display it on the web application.

**Comment by Ncode-93** [comment]
Hey @Sohammhatre10 I can work on this issue.

**Comment by Sohammhatre10** [comment]
Will assign you both @abhirajadhikary06 and @Ncode-93 this.

**Comment by abhirajadhikary06** [comment]
Thanks for assignig, Working to bring these feature live 👍

**Comment by pokymono** [comment]
I would like to contribute to this issue, pls assign me

**Comment by Ncode-93** [comment]
Thanks @Sohammhatre10 for assigning this issue

**Comment by Sohammhatre10** [comment]
@Ncode-93 @Rishitshivam @abhirajadhikary06 @Ankitv003 Any updates?

**Comment by pokymono** [comment]
I'll update you on this issue by tonight

**Comment by pokymono** [comment]
Hey, I have a quick question. What input should we collect from users for prediction and analysis? Should we include details like exam type, gender, name, rank, and preferred program? Also, what exactly should our predictive analysis focus on? Should we predict the number of colleges they could be eligible for or the specific branches they might qualify for?

**Comment by abhirajadhikary06** [comment]
> Hey, I have a quick question. What input should we collect from users for prediction and analysis? Should we include details like exam type, gender, name, rank, and preferred program? Also, what exactly should our predictive analysis focus on? Should we predict the number of colleges they could be eligible for or the specific branches they might qualify for?

From my P.O.V students predict college on their expected rank , So ask for the rank and using predictive analysis output the college they may get

**Comment by Sohammhatre10** [comment]
> > Hey, I have a quick question. What input should we collect from users for prediction and analysis? Should we include details like exam type, gender, name, rank, and preferred program? Also, what exactly should our predictive analysis focus on? Should we predict the number of colleges they could be eligible for or the specific branches they might qualify for?
> 
> From my P.O.V students predict college on their expected rank , So ask for the rank and using predictive analysis output the college they may get

@Rishitshivam @abhirajadhikary06 non inclusion of additional features like gender caste may hamper the accuracy so using the entire dataset as a context is necessary.
